docs/overview.md

```diff
@@ -0,0 +1,34 @@
+[◀︎ Airlift](../README.md)
+
+## Overview
+
+Airlift is a toolkit and not a framework. Frameworks such as Spring are very heavy-weight and will control the future of your project. Libraries, on the other hand, are small tools that enable your project by providing targeted utility. Think of Airlift as plumbing for your service. Airlift helps you by eliminating the drudgery of piecing together the basics while allowing you to focus on the functionality of your service.
```

Comment:
The third sentence still refers to libraries, so it sounds disconnected from the introduction.

Reply:
done